### PR TITLE
Simplify seqan3/std/concepts

### PR DESCRIPTION
**Pull request by marehr**
Fixes https://github.com/seqan/product_backlog/issues/190

### CODE REVIEW

File: include/seqan3/std/concepts [hunk 1]
```diff
@@ -64,81 +64,100 @@ namespace std
...
 using ::concepts::unsigned_integral;
-
-/*!\interface std::floating_point <>
- * \brief The concept std::floating_point<T> is satisfied if and only if T is a floating-point type.
- * \sa https://en.cppreference.com/w/cpp/types/is_floating_point
- */
+// using ::concepts::floating_point;
 //!\cond
 template <typename t>
 SEQAN3_CONCEPT floating_point = std::is_floating_point_v<t>;
```
Comment (smehringer) [inline include/seqan3/std/concepts]: why did you remove the documentation fthe floating_point concept although we still need to implement it?

Comment (marehr) [reply include/seqan3/std/concepts]: Because std::floating_point should use the external dokumentation of cppreference.
